conf/modules.config

```
@@ -467,7 +467,8 @@ process {
         publishDir = [
             path: { "${params.outdir}/kraken2/${meta.db_name}/" },
             mode: params.publish_dir_mode,
-            pattern: '*.{txt,fastq.gz}'
+            pattern: '*.{txt,fastq.gz}',
+            saveAs: {  params.bracken_save_intermediatekraken2 == false && meta.tool == bracken ? false : true  }
```

```suggestion
            saveAs: {  params.bracken_save_intermediatekraken2 == false && meta.tool == bracken ? null : it  }
```

Sorry, forgot about that 😅. The true/false applies when it's under `enabled` save has to be a name.

At least I hope so, comparing to the adapterremoval thing above